datafusion/functions/src/string/octet_length.rs

```diff
@@ -81,12 +81,6 @@ impl ScalarUDFImpl for OctetLengthFunc {
                 ScalarValue::Utf8(v) => Ok(ColumnarValue::Scalar(ScalarValue::Int32(
                     v.as_ref().map(|x| x.len() as i32),
                 ))),
-                ScalarValue::LargeUtf8(v) => Ok(ColumnarValue::Scalar(
```

Comment:
I like how removing the detailed physical types on the ScalarValue makes this kind of code much simpler (it doesn't heed to also handle two different "types" of strings